actioner/manifest.json

```
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "22996056-6abc-4d33-be11-3e27626b8a26",
```

```suggestion
  "app_uuid": "ca57fbd4-da6a-456a-b0ac-476fafd93373",
```
Update to match oauth client